superset/db_engine_specs/base.py

```diff
@@ -1304,6 +1304,28 @@ def get_column_spec(
             )
         return None
 
+    @classmethod
+    def get_cancel_query_payload(cls, cursor: Any, query: Query) -> Any:
+        """
+        Returns None if query can not be cancelled.
+        :param cursor: Cursor instance in which the query will be executed
+        :param query: Query instance
+        :return: Type of the payload can vary depends on databases
+        but must be jsonable. None if query can't be cancelled.
```

Comment:
> None if query can't be cancelled

could possibly be implied by better type hints.

Reply:
addressed by ef8436b783a2baac74322ff2b98c0189edc332e6